packages/vscode-extension/src/commands/UpdateObject.ts

```diff
@@ -31,19 +34,36 @@ export default class UpdateObject extends BaseAction {
 
 		const continueMessage = this.translationService.getMessage(ANSWERS.CONTINUE);
 		const cancelMessage = this.translationService.getMessage(ANSWERS.CANCEL);
-		const override = await window.showQuickPick([continueMessage, cancelMessage], {
-			placeHolder: this.translationService.getMessage(UPDATE_OBJECT.OVERRIDE, scriptId),
+		const yes = this.translationService.getMessage(ANSWERS.YES);
+		const no = this.translationService.getMessage(ANSWERS.NO);
+		const processCanceledMessage = this.translationService.getMessage(UPDATE_OBJECT.PROCESS_CANCELED);
+		const suiteCloudCommand = { scriptid: [scriptId], includeinstances: '' };
```

Comment:
I would prefer to name this one `suiteCloudCommandOptions` / `commandOptions` or similar one with "options" in the name.